FSiAdminSecurityBundle.php

```diff
@@ -43,6 +45,22 @@ public function build(ContainerBuilder $container)
         }
     }
 
+    public function boot()
+    {
+        /* @var $doctrine RegistryInterface */
+        $doctrine = $this->container->get('doctrine');
+        $userClass = $this->container->getParameter('admin_security.model.user');
+        $userRepositoryClass = 'FSi\Bundle\AdminSecurityBundle\Security\User\UserRepositoryInterface';
+        $repository = $doctrine->getRepository($userClass);
```

Comment:
This will only work if the returned metadata is of class `Doctrine\ORM\Mapping\ClassMetadataInfo`, since that field only exists there. Is it safe to make such assumption?